samples/apps/copilot-chat-app/webapi/CopilotChat/Models/ChatMessage.cs

```diff
@@ -150,7 +151,14 @@ public static ChatMessage CreateBotResponseMessage(string chatId, string content
     /// <returns>A formatted string</returns>
     public string ToFormattedString()
     {
-        return $"[{this.Timestamp.ToString("G", CultureInfo.CurrentCulture)}] {this.UserName}: {this.Content}";
+        var content = this.Content;
+        if (this.Type == ChatMessageType.Document)
+        {
+            var documentDetails = JsonSerializer.Deserialize<DocumentMessageContent>(content);
```

Comment:
I don't think so? not unless I make define another field to specifically store a `DocumentMessageContent`, which I wanted to avoid.

otherwise, we need to serialize the struct when saving it to `Content` and then de-serialize it when we use `Content` in `ToFormattedString()`. am I missing something here you think?

Reply:
I was thinking about storing a DocumenetMessageContent too. It can be null when the message is not a document message. Since you wanted to avoid that, I think we are all good!